scipy/stats/_distn_infrastructure.py

```diff
@@ -1202,6 +1194,11 @@ def stats(self, *args, **kwds):
                     if mu2 is None:
                         mu2p = self._munp(2, *goodargs)
                         mu2 = mu2p - mu * mu
+                    if g1 is None:
+                        mu3 = None
+                    else:
+                        # (mu2**1.5) breaks down for nan and inf
```

Comment:
I don't think this is true anymore (if it ever was). In that case, all of this could be:
```python3
mu3 = g1 * mu2**1.5 if g1 is not None else None
```

Reply:
Yes, makes sense